Makefile

```diff
@@ -71,7 +71,7 @@ e2e-sanity: e2e-sanity-ovirt e2e-sanity-vsphere
 
 e2e-sanity-ovirt:
 	# ovirt suite
-	go test ./tests/suit -v -ginkgo.focus ".*oVirt.*"
+	KUBEVIRT_CLIENT_GO_SCHEME_REGISTRATION_VERSION=v1 go test ./tests/suit -v -ginkgo.focus ".*oVirt.*"
```

Comment:
why we need it now?

Reply:
because we query a KubeVirt entity (the imported VM), see [this](https://github.com/kubevirt/kubevirt/pull/4749)